src/main/java/com/jfrog/xray/client/impl/services/scan/GraphResponseImpl.java

```diff
@@ -32,6 +33,12 @@ public void setPackageType(String packageType) {
         this.packageType = packageType;
     }
 
+    @Override
+    @JsonProperty("progress_percentage")
+    public int getProgressPercentage() {
```

Comment:
Why int and not double?

Reply:
Because this is an integer number between 0 and 100